Getting-Started/Setup/Requirements/index-v8.md

```diff
@@ -0,0 +1,27 @@
+---
+versionFrom: 8.0.0
+---
+
+# System Requirements
+
+## Browsers
+The Umbraco UI should work in all modern browsers:
+
+* Firefox (Latest)
+* Chrome (Latest)
+* Safari (Latest)
+* Edge
+
+## Local Development
+* Microsoft Windows 7 SP1
+* [Visual Studio Code](https://code.visualstudio.com/) with the [IISExpress extension](https://marketplace.visualstudio.com/items?itemName=warren-buckley.iis-express) or [Microsoft Visual Studio](https://www.visualstudio.com/) 2017 **version 15.9.7**
```

Comment:
```suggestion
* [Visual Studio Code](https://code.visualstudio.com/) with the [IISExpress extension](https://marketplace.visualstudio.com/items?itemName=warren-buckley.iis-express) or [Microsoft Visual Studio](https://www.visualstudio.com/) 2017 **version 15.9.x+**
```
The issue was fixed in 15.9.0: https://developercommunity.visualstudio.com/content/problem/315361/systemruntime-412-assembly-binding-redirects.html